### DISCUSSION TITLE
Creating Exhibit Book/Catalog

### DISCUSSION THREAD

**Discussion by drbobbyduke** [thegetty/quire#914] [General]
What is the process for getting books printed once the print version has been exported?  Find a local printer?  For the Getty publications that people can purchase are they printed once purchased online or does Getty keep a stock somewhere?

**Comment by Erin-Cecele** [comment]
Hi @drbobbyduke, at Getty, we do short-run digital printing of a couple hundred copies and reprints when we run out of those. Because of the non-Quire books Getty publishes, we already have a printer/distributor that we work with. Other Quire users, especially those without large publishing programs like Getty, tend to go the self-publishing route. For example, Amon Carter Museum used Blurb.com: https://www.blurb.com/bookstore/invited/9733129/47ed827d5e3ca2bc10f2c86e12400f1307f2a28e and North Western University Libraries used Lulu.com: https://www.lulu.com/shop/hirokazu-miyazaki/nuclear-compensation/paperback/product-jm9n4j.html?page=1&pageSize=4. We've also heard good things about Book Baby: https://www.bookbaby.com/.

**Comment by Erin-Cecele** [comment]
We'd love to share your project with our community when it's complete. If you're interested in having your project featured on the [Quire Community Showcase](https://quire.getty.edu/community/community-showcase/) or in an upcoming newsletter, please feel free to complete this submission form: https://docs.google.com/forms/d/e/1FAIpQLSeJWyTA9OUjdQnw8JsmJudJuIzGRAzTeTL7BWr_L6u6C67bCA/viewform.
